inst/report/Report_KRI.Rmd

```diff
@@ -74,17 +74,33 @@ if(setup$GroupLevel == "Site"){
 # subset data on latest snapshot date
 dfResults_current <- FilterByLatestSnapshotDate(params$dfResults)
 
-htmltools::div(
-  style = "table-layout: fixed; width: 100%; overflow: auto;",
-  Widget_GroupOverview(
-      dfResults = dfResults_current,
-      dfGroups = params$dfGroups,
-      dfMetrics = params$dfMetrics, 
-      strGroupLevel = setup$GroupLevel,
-      strGroupLabelKey = GroupLabelKey
-  )
+GroupOverViz <- Widget_GroupOverview(
+  dfResults = dfResults_current,
+  dfGroups = params$dfGroups,
+  dfMetrics = params$dfMetrics, 
+  strGroupLevel = setup$GroupLevel,
+  strGroupLabelKey = GroupLabelKey
 )
 
+# Remove sites whose metrics have no flags 
+dfResults_atleastoneflag <- params$dfResults %>% 
```

Comment:
@lauramaxwell @jwildfire Do we want to turn this snippet into some sort of utility function of some sort that filters out the no-flag rows? Wasn't sure how much data manipulation was appropriate, if any, inside the `Report_KRI.Rmd` was

Reply:
@zdz2101 yes, a utility function to handle this filter would be best. Want to minimize custom data manipulation inside the Rmd when possible.